### DISCUSSION TITLE
How to use parameters (-p) argument?

### DISCUSSION THREAD

**Discussion by eiglow** [lightspark/lightspark#765] [Q&A] [answered]
It says that a "params-file" should follow, but how does this work? Like, what format should the file be in?

**Comment by dbluelle** [comment, marked as answer]
the manpage says 
"Load flash parameters from file. Every odd line will be interpreted as a parameter name, with the following one as the value."
so that means it is a simple text file where the first line contains the key and the second line the corresponding value and so on.
Example:
http://www.example.com/test.swf?var1=a&var2=b&var3=c
can be translated into a params file like this:
```
var1
a
var2
b
var3
c
```
The same mechanism can also be used for variables set to the flash tag in the html code of the web page.
